demos/python_demos/multi_camera_multi_person_tracking/tools/run_evaluate.py

```diff
@@ -0,0 +1,151 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import argparse
+import json
+
+import numpy as np
+import motmetrics as mm
+from lxml import etree
+from tqdm import tqdm
+import glog as log
```

Comment:
The other demos use the built-in `logging` module for logging. Please do the same.

Reply:
Fixed.